test/spec/config/env/development.spec.js

```diff
@@ -7,16 +7,16 @@
             'util/container',
             'config/env/development'
         ],
-        function (chai, container, development) {
+        function (chai, container, configure) {
             var expect = chai.expect;
 
             describe('The `config/env/development` module', function () {
                 it('Defines a function', function () {
-                    expect(development).to.be.a('function');
+                    expect(configure).to.be.a('function');
                 });
                 describe('When invoked', function () {
                     beforeEach(function () {
-                        development();
+                        configure();
```

Comment:
And here's the reason for it to not be called `configuration` - ignore previous comment.